runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/FakeWindmillServer.java

```diff
@@ -367,6 +375,9 @@ public boolean commitWorkItem(
         errorCollector.checkThat(
             request.getShardingKey(), allOf(greaterThan(0L), lessThan(Long.MAX_VALUE)));
         errorCollector.checkThat(request.getCacheToken(), not(equalTo(0L)));
+        // Throw away per work item latency attributions because they are not deterministic in tests
+        // for valid comparison.
+        request = request.toBuilder().clearPerWorkItemLatencyAttributions().build();
```

Comment:
ditto (or perhaps just here in the stream case since we're not doing this for appliance)

Reply:
Done.